.babelrc

```diff
@@ -1,3 +1,11 @@
 {
-  "presets": ["es2015", "stage-2"]
+  "presets": ["stage-2", ["es2015", { "modules": false }]],
```

Comment:
does modules: false not break based on the fact that we use es modules in the src/? Also I am curious why is modules: false necessary to get the umd build working?